Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -199,6 +199,10 @@ darker input fields:           #505050
         width: 14px;
     }
 
+    /* ZEN-30116 background changes for panel header */
+    .z-cse .x-panel-header-default {
+        background-color: #f3f3f3;
```

Comment:
this line overrides the previous rule for **x-panel-header-default** on line 152.  should we remove line 152?

Reply:
this line override only **background-color**, and we have no need to remove line 152 because there we override original ExtJs css **background-image: none;**

Reply:
Fair enough. I'll accept that answer.